extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -58,6 +62,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     const rawJson = this.element.getAttribute('json');
     const jsonParameters = rawJson ? JSON.parse(rawJson) : {};
     const tfcd = jsonParameters['tfcd'];
+    const adTestOn = isInExperiment(this.element, MANUAL_EXPERIMENT_ID);
```

Comment:
Sure, by exposing a specific `#isInManualExperiment` predicate.  That's a bit specialized, but okay I guess.  Done.